src/code/z_sram.c

```diff
@@ -856,7 +857,7 @@ void Sram_EraseSave(FileSelectState* fileSelect, SramContext* sramCtx) {
     MemCpy(sramCtx->readBuff + offset, &gSaveContext, sizeof(Save));
     SsSram_ReadWrite(OS_K1_TO_PHYSICAL(0xA8000000) + offset, &gSaveContext, SLOT_SIZE, OS_WRITE);
 
-    PRINTF("ＣＬＥＡＲ終了\n");
+    PRINTF(("ＣＬＥＡＲ終了\n", "CLEAR END\n"));
```

Comment:
forgor the T, this is why it failed
```suggestion
    PRINTF(T("ＣＬＥＡＲ終了\n", "CLEAR END\n"));
```